lib/sda/processor.go

```diff
@@ -0,0 +1,50 @@
+package sda
```

Comment:
Can you put experimental things in `services_scratch`? I would be very happy to use your `services2` as soon as possible. We need a service for `SkipChains` and `RandHound`. We can do them using my simple implementation, but if we don't have to rewrite the services once `services2` is done, that would be even more cool.